docs/settings.md

```diff
@@ -60,6 +60,31 @@ output_matplotlib_strings
     - `"remove-warn"` (default) or `"remove-error"`: remove all matplotlib strings in outputs, and log a warning or error
     - `"warn"` or "error": log a warning or error if matplotlib strings in outputs
 
+## Markdown Parsing Settings
+
+Adding an object of `parser` to the settings will allow you to control various parser behaviors.
+
+```{code-block} yaml
+:filename: myst.yml
+project:
+  settings:
+    parser:
+      dollarmath: false
+```
```

Comment:
This is the config that I have added, it is in the `project.settings.parser` right now just enabling/disabling dollarmath, but many other options could easily come in here as well.